recipe/meta.yaml

```diff
@@ -3,7 +3,7 @@
 {% set zlib_api = "1.3.1" %}
 # Please do not reset the build number unless zlib_api changed. It needs to be
 # incremental across both outputs using version and these using zlib_api.
-{% set number = 1 %}
+{% set number = 2 %}
 
 package:
   name: zlib-ng
```

Comment:
You should rename this to `zlib-ng-split` as the top level output name has to be an output that doesn't depend on any of the other outputs for `pin_subpackage` to work.